src/zenml/config/server_config.py

```diff
@@ -206,6 +206,21 @@ class ServerConfiguration(BaseModel):
             one of the reserved values `disabled`, `no`, `none`, `false`, `off`
             or to an empty string, the `Permissions-Policy` header will not be
             included in responses.
+        server_name: The name of the ZenML server. Used only during initial
+            deployment. Can be changed later as a part of the server settings.
+        display_announcements: Whether to display announcements about ZenML in
+            the dashboard. Used only during initial deployment. Can be changed
+            later as a part of the server settings.
+        display_updates: Whether to display notifications about ZenML updates in
+            the dashboard. Used only during initial deployment. Can be changed
+            later as a part of the server settings.
+        auto_create_default_user: Whether to automatically create a default
+            admin user account with an empty password. Used only during initial
+            deployment.
+        auto_activate: Whether to automatically activate the ZenML server during
+            deployment. Used only during initial deployment. If
+            `auto_create_default_user` is set to `True`, this setting is ignored
+            and the server is activated automatically.
```

Comment:
Can you explain why they are different? When I want to use `auto_activate` instead of `auto_create_default_user`?

Reply:
you're probably right, it is overkill to have two configuration options. I'll remove the `auto_activate` one.